sql/rpl_info_table.cc

```diff
@@ -21,6 +21,46 @@
 #include "rpl_info_values.h"        // Rpl_info_values
 #include "sql_class.h"              // THD
 
+class Thd_local_mem_root
```

Comment:
Make this class non-copyable (add private undefined copy constructor and assignment operator).